uwblueprint/community-fridge-kw
# feat: send donation cancellation email

## Brief description. What is this change? 

### [Send Donation Cancellation Email ](https://www.notion.so/uwblueprintexecs/Send-Donation-Cancellation-Email-6038157f70d64d65acc1357be19a647c)



## Implementation description. How did you make this change?
Admin email (recurring donation)
![Screen Shot 2022-02-12 at 4 00 42 PM](https://user-images.githubusercontent.com/38363056/153728378-9656f2e8-a8f5-4105-8023-e6d92ffa4cfd.png)
Donor email (recurring donation)
![Screen Shot 2022-02-12 at 4 00 27 PM](https://user-images.githubusercontent.com/38363056/153728390-7da6b9c4-1999-41fa-973f-63b7fdd97efa.png)

Note: please refer to Notion ticket for copy for when **admin deletes donations**

* added `sendEmailVerificationAfterSchedulingADonation` to scheduling service
* call `sendEmailVerificationAfterSchedulingADonation` in `deleteSchedulingById` and `deleteSchedulingByRecurringDonationId`



## Steps to test
1. After checking out branch, in the implementation of `schedulingService.ts`, **replace `TEMP_ADMIN_EMAIL` with YOUR personal email** so we don't spam the actual community fridge account :)
2. Create a one-time donation
2. Cancel it
4. Ensure email is received to donor email (refer to copy on linked Notion ticket)
5. Do steps 2-4 with a recurring donation
6. Do steps 2-5 where admin (account using your uwblueprint email) deletes the schedules




## Checklist
